www/themes/custom/drupalcampcebu2015/public/sass/partials/_sponsors.scss

```diff
@@ -84,7 +84,8 @@ Temporary fix, display: none. */
 /* Sponsor header text */
 .sponsor-header {
   padding: 3em 1.6em;
-  overflow: hidden;
+  /* Possible fix for the bug in sponsor tabs background-color */
```

Comment:
This didnt really fixed the bug. After this changed had been done, the bug still appeared during the hackathon, already mentioned thsi to @leolandotan